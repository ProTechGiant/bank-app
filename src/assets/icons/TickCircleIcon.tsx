import { Path, Svg } from "react-native-svg";

import { IconProps } from "@/assets/icons";

export function TickCircleIcon({ color = "#F34C33", height = 80, width = 80 }: IconProps) {
  return (
    <Svg width={width} height={height} viewBox="0 0 80 80" fill="none" xmlns="http://www.w3.org/2000/svg">
      <Path
        d="M40 0C17.92 0 0 17.92 0 40C0 62.08 17.92 80 40 80C62.08 80 80 62.08 80 40C80 17.92 62.08 0 40 0ZM32 60L12 40L17.64 34.36L32 48.68L62.36 18.32L68 24L32 60Z"
        fill={color}
      />
    </Svg>
  );
}
