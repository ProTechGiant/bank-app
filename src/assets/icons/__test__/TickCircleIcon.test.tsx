import { create } from "react-test-renderer";

import { TickCircleIcon } from "../TickCircleIcon";

describe("icon/TickCircleIcon", () => {
  it("renders a TickCircleIcon", () => {
    const result = create(<TickCircleIcon />);

    expect(result).toMatchSnapshot();
  });
});
