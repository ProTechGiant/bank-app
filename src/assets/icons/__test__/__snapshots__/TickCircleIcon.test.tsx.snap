// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon/TickCircleIcon renders a TickCircleIcon 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight="80"
  bbWidth="80"
  fill="none"
  focusable={false}
  height={80}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    [
      {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      {
        "flex": 0,
        "height": 80,
        "width": 80,
      },
    ]
  }
  vbHeight={80}
  vbWidth={80}
  width={80}
  xmlns="http://www.w3.org/2000/svg"
>
  <RNSVGGroup
    fill={null}
    propList={
      [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M40 0C17.92 0 0 17.92 0 40C0 62.08 17.92 80 40 80C62.08 80 80 62.08 80 40C80 17.92 62.08 0 40 0ZM32 60L12 40L17.64 34.36L32 48.68L62.36 18.32L68 24L32 60Z"
      fill={
        {
          "payload": 4294134835,
          "type": 0,
        }
      }
      propList={
        [
          "fill",
        ]
      }
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
